app/data_providers/dispatch.py

```diff
@@ -171,39 +175,51 @@ async def get_slate_lineup(
     async def get_en_us_slate_lineup(
             self, user: RequestUser, recommendation_count: int, locale: LocaleModel
     ) -> Tuple[CorpusSlateLineupModel, Optional[UnleashAssignmentModel]]:
-
         """
         :param user:
         :param recommendation_count: Maximum number of recommendations to return.
         :param locale:
         :return: Slate lineup for en-US Home:
-            1. 'For You' slate if preferred topics are available, or otherwise 'Recommended Reads'
-            2. Pocket Hits
-            3. Collection slate
-            4. 'Life Hacks' slate
-            5. Topic slates according to preferred topics if available, otherwise default topics.
+
+                If in the new v3 test
+                    1. Syndicated - Pocket Worthy
+                    2. Pocket Hits
+                    3. 'For You' slate if preferred topics are available, or otherwise 'Recommended Reads'
+                    4. 'Life Hacks' slate
+                Otherwise:
+                    1. 'For You' slate if preferred topics are available, or otherwise 'Recommended Reads'
+                    2. Pocket Hits
+                    3. Collection slate
+                    4. 'Life Hacks' slate
         """
+
         user_impression_capped_list, \
-        preferred_topics = await gather(
+            preferred_topics = await gather(
             self.user_impression_cap_provider.get(user),
             self._get_preferred_topics(user))
 
         slates = []
-        if preferred_topics:
-            slates += [self.for_you_slate_provider.get_slate(
-                preferred_topics=preferred_topics,
-                user_impression_capped_list=user_impression_capped_list
-            )]
+        assignment = await self.unleash_provider.get_assignment(POCKET_HOME_V3_FEATURE_FLAG, user=user)
```

Comment:
If you put this in the above `gather` call it will happen asynchronously with fetching the topics and impression caps. For example: https://github.com/Pocket/recommendation-api/pull/1078/files#diff-acaa726ac78ef290df20b378dad8f436fdb4f3d21b675a0ba561f76a03c0ed3cL198